lib/auth/accountrecovery_test.go

```diff
@@ -402,10 +374,6 @@ func TestVerifyAccountRecovery_WithAuthnErrors(t *testing.T) {
 			require.True(t, trace.IsAccessDenied(err))
 			require.Equal(t, verifyRecoveryBadAuthnErrMsg, err.Error())
 
-			attempts, err := srv.Auth().GetUserRecoveryAttempts(ctx, u.username)
-			require.NoError(t, err)
-			require.Len(t, attempts, 1)
```

Comment:
Should we keep a modified version of this assertion to ensure that attempts are still recorded, or that covered elsewhere?

Reply:
With this change there was no need to retrieve the attempts.  So eventually I expected this API will be removed from the Auth.  I considered trying to add in an API just for test, but I don't feel this check adds much value.  The general account behaviors are already covered.